src/components/list/examples/list-custom-component.tsx

```diff
@@ -0,0 +1,86 @@
+import { ListItem } from '@limetech/lime-elements';
```

Comment:
I think this commit could be part of the commit that adds the feature, but keeping it separate is ok too. _But_, if you keep it separate, the commit type should be `docs`, not `feat`:

`docs(list): create a list example with a custom component`